CHANGELOG.md

```diff
@@ -22,6 +22,12 @@
 - [#4063](https://github.com/firecracker-microvm/firecracker/pull/4063):
   Adds source-level instrumentation based tracing. See
   [tracing](./docs/tracing.md) for more details.
+- [#4226](https://github.com/firecracker-microvm/firecracker/pull/4226):
+  Added support for vhost-user device metrics. Each vhost-user device will
+  emit metrics under the label `"vhost_user_{device}_{drive_id}"`.
+  E.g. the associated metrics for vhost-user block device with endpoint
+  `"/drives/rootfs"` will be available under `"vhost_user_block_rootfs"`
+  in the metrics json object.
```

Comment:
Maybe this should be in the #4210? @kalyazin

Reply:
good point, added